config/evss/mock_letters_response.yml.example

```diff
@@ -412,11 +412,11 @@ default:
         has_survivors_pension_award: false
         monthly_award_amount: 123.0
         service_connected_percentage: 2
-      has_adapted_housing: false
-      has_chapter35_eligibility: true
-      has_death_result_of_disability: false
-      has_individual_unemployability_granted: false
-      has_special_monthly_compensation: false
+        has_adapted_housing: false
```

Comment:
Looks like it doesn't exist for this user only.  I would imagine that this may be intentional for some test... I'm hesitant to change it since it's just test data.